### ISSUE TITLE
Plugins loaded from an npm namespaced package (@namespace/project) fail to load

### ISSUE THREAD

**Issue by weotch** [nuxt/nuxt.js#3648]
### Version

[v1.4.2](https://github.com/nuxt.js/releases/tag/v1.4.2)

### Reproduction link

[https://github.com/weotch/nuxt-namespaced-plugin-bug](https://github.com/weotch/nuxt-namespaced-plugin-bug)

### Steps to reproduce

Run `yarn build`.

### What is expected ?

The build to succeed.

### What is actually happening?

```
ERROR in ./.nuxt/index.js
Module not found: Error: Can't resolve 'nuxt_plugin_utils_9f9d6c7a' in '/Users/reinhard/Work/Open Source/nuxt-namespaced-plugin-bug/.nuxt'
 @ ./.nuxt/index.js 200:0-68
 @ ./.nuxt/client.js

 ERROR 

  Error: Webpack build exited with errors
  
  - builder.js:589 compiler.run
    [nuxt-namespaced-plugin-bug]/[nuxt]/lib/builder/builder.js:589:29
  
  - Compiler.js:265 emitRecords.err
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/Compiler.js:265:13
  
  - Compiler.js:371 Compiler.emitRecords
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/Compiler.js:371:38
  
  - Compiler.js:258 emitAssets.err
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/Compiler.js:258:10
  
  - Compiler.js:364 applyPluginsAsyncSeries1.err
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/Compiler.js:364:12
  
  - Tapable.js:218 next
    [nuxt-namespaced-plugin-bug]/[tapable]/lib/Tapable.js:218:11
  
  - SizeLimitsPlugin.js:99 Compiler.compiler.plugin
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/performance/SizeLimitsPlugin.js:9    9:4
  
  - Tapable.js:222 Compiler.applyPluginsAsyncSeries1
    [nuxt-namespaced-plugin-bug]/[tapable]/lib/Tapable.js:222:13
  
  - Compiler.js:361 Compiler.afterEmit
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/Compiler.js:361:9
  
  - Compiler.js:350 require.forEach.err
    [nuxt-namespaced-plugin-bug]/[webpack]/lib/Compiler.js:350:15
  
  - async.js:473 
    [nuxt-namespaced-plugin-bug]/[async]/dist/async.js:473:16
  
  - async.js:1064 iteratorCallback
    [nuxt-namespaced-plugin-bug]/[async]/dist/async.js:1064:13
  
  - async.js:969 
    [nuxt-namespaced-plugin-bug]/[async]/dist/async.js:969:16
  
  - graceful-fs.js:43 
    [nuxt-namespaced-plugin-bug]/[graceful-fs]/graceful-fs.js:43:10
  
  - fs.js:145 FSReqWrap.oncomplete
    fs.js:145:20
```

### Additional comments?

I suspect that Nuxt's `@` aliases (https://nuxtjs.org/guide/directory-structure) are interfering here, obscuring how to resolve the namespaced package name.  When I make reference to the file within `node_modules` (https://github.com/weotch/nuxt-namespaced-plugin-bug/blob/master/nuxt.config.js#L6) the build _does_ succeed.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c7510">#c7510</a>)</em></sub></div>

**Comment by weotch** [comment]
In case it wasn't clear, the only things I've changed after doing a standard vue-cli install is this bit: https://github.com/weotch/nuxt-namespaced-plugin-bug/blob/master/nuxt.config.js#L3-L7.

**Comment by Atinux** [comment]
@clarkdo, any idea about this?

**Comment by clarkdo** [comment]
I think its related to 
https://github.com/nuxt/nuxt.js/blob/3f1d634fb7f43676090a873ba208ded1f9ce6be1/lib/builder/builder.js#L73, I’ll figure it out tmr.

**Comment by clarkdo** [comment]
@weotch This issue has been fixed in nuxt-edge (Nuxt 2.0 will be published shortly) and I have submitted a pr to your repo for upgrading nuxt, so I'll close the issue now.

**Comment by weotch** [comment]
Thanks!  Confirmed that your PR fixed it.

> Nuxt 2.0 will be published shortly

Very excited!!
